batch_job/src/work_queue_factory.c

```diff
@@ -316,14 +317,33 @@ static void set_worker_resources_options( struct batch_queue *queue )
 	resource_args = xxstrdup(buffer_tostring(&b));
 	buffer_free(&b);
 }
-
+static char* fetch_end_of_path(char* path){
```

Comment:
We usually use fetch for transfers over the network. 

Also, see:  dttools/src/path.h:path_basename